src/app/components/MediaLoader/utils/buildSettings.test.ts

```diff
@@ -167,7 +167,7 @@ describe('buildSettings', () => {
           },
           enableToucan: true,
           externalEmbedUrl:
-            '/ws/av-embeds/cps/serbian/lat/srbija-68707945/p01k6msp/sr-latn',
+            'https://www.test.bbc.com/serbian/lat/av-embeds/srbija-68707945/vpid/p01k6msp',
```

Comment:
These are more accurate in my mind. `/ws/av-embeds` should only be for our AMP pages when it comes to CPS assets. The `<service>/av-embeds/` route is used for external embedding, so this feels better.